impeller/typographer/backends/skia/text_render_context_skia.cc

```diff
@@ -104,10 +105,27 @@ static size_t OptimumAtlasSizeForFontGlyphPairs(
 }
 
 static std::shared_ptr<SkBitmap> CreateAtlasBitmap(const GlyphAtlas& atlas,
-                                                   size_t atlas_size) {
+                                                   size_t atlas_size,
+                                                   bool* has_color) {
```

Comment:
Rather than having this be an out param, we could figure it out based on the colorType of the bitmap.

Reply:
I put this back into the atlas, because I need to track each font + glyph pair anyway